api/python/quilt3/packages.py

```diff
@@ -1229,6 +1231,15 @@ def physical_key_is_temp_file(pk):
             self._set(lk, pkg[lk])
 
         pkg._build(name, registry=registry, message=message)
+
+        navigator_url = get_from_config("navigator_url")
+        registry_parsed = PhysicalKey.from_url(registry)
+        shorthash = Package._shorten_tophash(name, registry_parsed, pkg.top_hash)
+        print(f"Package {name}@{shorthash} pushed to s3://{dest_parsed.bucket}")
+        if navigator_url is None:
+            print(f"Run `quilt3 catalog {str(dest_parsed)}` to browse.")
```

Comment:
I think this line will send the browser to the wrong catalog page (/b/BUCKET/tree/...) instead of /b/BUCKET/packages/USER/PACKAGE).